cookbook/core/control_flow/chain_tasks.py

```diff
@@ -41,13 +38,12 @@ def write():
             "species": ["setosa"],
         }
     )
-    # we write the data to a database
-    # pd.to_csv("...")
-
 
+    # Write the data to a database
+    # pd.to_csv("...")
 # %%
-# We want to enforce an order here: ``write()`` followed by ``read()``.
-# Since no data-passing happens between the tasks, we use ``>>`` operator on the nodes.
+# We want to enforce an order here: ``write()`` followed by ``read()``. Since no data-passing happens between the tasks, we use ``>>`` operator on the nodes.
+
```

Comment:
We don't need a new line between the content and code.

Reply:
Done